packages/test-app/common/utils.ts

```diff
@@ -15,18 +16,21 @@ export const getAuthOrigin = () => {
     // "stacks-authenticator" for the authenticator.
     authOrigin = document.location.origin.replace('authenticator-demo', 'stacks-authenticator');
   } else if (origin.includes('authenticator-demo')) {
-    // TODO: revert this when 301 is merged
-    authOrigin = 'https://deploy-preview-301--stacks-authenticator.netlify.app';
-    // authOrigin = 'https://app.blockstack.org';
+    authOrigin = 'https://app.blockstack.org';
   }
   return authOrigin;
 };
 
-export const getRPCClient = () => {
-  const { origin } = location;
-  const url = origin.includes('localhost')
-    ? 'http://localhost:3999'
-    : 'https://stacks-node-api.blockstack.org';
+export const useLocalNode = location.origin.includes('localhost') && false;
```

Comment:
Is there a certain behavior required for testing?

Reply:
Yes, when running the test app locally, I'd like to default to the local node URL.